packages/amplication-client/package.json

```
@@ -48,19 +44,19 @@
     "styled-components": "5.1.1",
     "use-debounce": "3.4.3",
     "xlsx": "0.16.9",
-    "yaml": "1.10.2",
-    "yup": "0.32.11"
+    "yaml": "1.10.2"
   },
   "scripts": {
-    "start": "react-app-rewired start",
+    "start": "craco start",
     "start:debug": "npm start",
-    "build": "react-app-rewired build",
-    "test": "react-app-rewired test",
+    "build": "CI=false BUILD_PATH=../../dist/apps/@amplication/client craco build",
```

`CI=false` will have precedence over any other ver definition of `CI`.
Is this on purpose? 

```suggestion
    "build": "BUILD_PATH=../../dist/apps/@amplication/client craco build",
```

---

@barshimi I think we should just get rid of these warnings. These are very trivial to fix and maybe this can even automatically fixed.

Anyway, let's tackle it after the deployment.